lib/world_cup.rb

```diff
@@ -0,0 +1,59 @@
+class WorldCup
+  attr_reader :year, :teams
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def active_players_by_position(position)
```

Comment:
This looks really good - one thing I would suggest is to break this into two methods to maintain SRP.  One method might return all active teams and the other could grab the players from those active teams.